kinto_http/session.py

```diff
@@ -1,12 +1,18 @@
 import time
-
+import sys
 import requests
 from six.moves.urllib.parse import urlparse
-
+import pkg_resources
 from kinto_http import utils
 from kinto_http.exceptions import KintoException, BackoffException
 
 
+kinto_http_version = pkg_resources.get_distribution("kinto_http").version
+requests_version = pkg_resources.get_distribution("requests").version
+python_version = '.'.join(map(str, sys.version_info[:3]))
```

Comment:
Another way of writing this:

`'{}.{}.{}'.format(*sys.version_info[:3])`

Maybe it's a matter of taste :)